### PR TITLE
Change python-yaml depend tag to use the Python 3 version. Simplifies ins…

### PR DESCRIPTION
**Pull request by tdenewiler**
…talling dependencies on Noetic.

Signed-off-by: Thomas Denewiler <thomas.denewiler@navy.mil>

### CODE REVIEW

**Comment by tdenewiler** [comment]
This change also helps on Ubuntu 20.04 with ROS2 Foxy.

File: uuv_control/uuv_control_utils/package.xml [hunk 1]
```diff
@@ -26,7 +26,8 @@
   <exec_depend>python3-numpy</exec_depend>
   <exec_depend>uuv_control_msgs</exec_depend>
   <exec_depend>std_msgs</exec_depend>
-  <exec_depend>python-yaml</exec_depend>
+  <exec_depend condition="$ROS_PYTHON_VERSION == 2">python-yaml</exec_depend>
```
Comment (jpliquid) [inline uuv_control/uuv_control_utils/package.xml:29]:
```suggestion
```

File: uuv_control/uuv_control_utils/package.xml [hunk 2]
```diff
@@ -26,7 +26,8 @@
...
+  <exec_depend condition="$ROS_PYTHON_VERSION == 3">python3-yaml</exec_depend>
```
Comment (jpliquid) [inline uuv_control/uuv_control_utils/package.xml:30]:
```suggestion
  <exec_depend>python3-yaml</exec_depend>
```

**Review by jpliquid — CHANGES_REQUESTED** [review]
Hi @tdenewiler!
Thank you for pointing out that incorrect dependency. Plankton does not support Noetic and Python 2.7, so testing the version is not required. I suggested the following changes. Could you update your commit please ?

**Review by jpliquid — APPROVED** [review]

**Comment by jpliquid** [comment]
Thanks for your contribution. I needed to change the title of the PR to reflect its content before merging.

**Pull request merged**